src/api/config/security.py

```
@@ -0,0 +1,85 @@
+from datetime import timedelta
+import os
+from dotenv import load_dotenv
+import secrets
+
+# Load environment variables
+load_dotenv()
+
+# Get absolute path for logs
+LOG_DIR = os.path.abspath(os.getenv('LOG_DIR', 'logs'))
+
+class SecurityConfig:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)